File: DailyFlash Solutions/DailyFlash Solutions/01-DailyFlash_Solutions/14_Jan_Solutions_Two/java/MaxFromThree.java
/*
Program 5: Write a Program to Find Maximum between three numbers
Input: 1 4 2
Output: 4 is Max number among 1, 4 & 2 

 */

/** DEFINATION OF CLASS MAXIMUM_AMONG_THREE*/
class MaximumAmongThree{

	/** DEFINATION OF THE MAIN FUNCTION*/
	public static void main(String args[]){


		int a=10000,b=200,c=3000;
		

		if(a > b && a> c){
			
			System.out.println("a is greater then b and c");
		}//END OF IF
		else if (b>a && b>c){
			System.out.println("b is greater then a and c");
		}//END OF ELSE IF
		else if (c>a && c>b){
			System.out.println("c is greater then b and a");
		}//END OF ELSE IF	



	}//END OF MAIN FUNCTION

}//END OF CLASS MAXIMUM_AMONG_THREE	

File: DailyFlash Solutions/DailyFlash Solutions/01-DailyFlash_Solutions/14_Jan_Solutions_Two/c/maxfromthree.c
/*
 *
Program 5: Write a Program to Find Maximum between three numbers
Input: 1 4 2
Output: 4 is Max number among 1, 4 & 2


*/


#include<stdio.h>

void main(){

	int a=1000,b=200,c=3000;

	if(a>b && a>c){

		printf("a = %d is greater than b = %d and c = %d",a,b,c);
	}else if(b>a && b>c){
		printf("b = %d is greater than a = %d  and c = %d",b,a,c);
	}else if(c>a && c>b){
		printf("c = %d is greater than a = %d  and b = %d",c,a,b);

	}
}
